notify/mute.go

```
@@ -51,6 +51,19 @@ type MuteFunc func(ctx context.Context, lset model.LabelSet) bool
 // Mutes implements the Muter interface.
 func (f MuteFunc) Mutes(ctx context.Context, lset model.LabelSet) bool { return f(ctx, lset) }
 
+// recordMuted adds the hashes of the given alerts to the set of muted alert
+// hashes in the context
```

golangci-lint (2.13.2)

[error] Comment should end in a period

(godot)